src/umpire/strategy/DynamicPoolList.cpp

```diff
@@ -122,10 +122,15 @@ bool DynamicPoolList::tracksMemoryUse() const noexcept
 void DynamicPoolList::coalesce() noexcept
 {
   UMPIRE_LOG(Debug, "()");
-  umpire::event::record([&](auto& event) {
```

Comment:
This umpire::event::record line should not be removed as it is needed for the `replay` tool.

Reply:
Will do!